packages/0xcert-scaffold/src/core/gateway.ts

```diff
@@ -309,11 +309,18 @@ export interface AssetLedgerData {
   symbol: string;
 
   /**
-   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended foo each asset.
+   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended for each asset.
    * Example: https://example.com/id/
-   * Asset 1 URI will become: https://example.com/id/1
+   * Asset 1 URI will become: https://example.com/id/1 + postfix
    */
-  uriBase: string;
+  uriPrefix: string;
+
+  /**
+   * Uri postfix for metadata URI-s. After uriPrefix and assetId postfix is automatically appended for each asset.
```

Comment:
* URI postfix for metadata URIs. After uriPrefix and assetId, postfix is automatically appended for each asset.